build.sbt

```diff
@@ -9,8 +9,8 @@ import UnidocKeys._
 import sbt.ScriptedPlugin._
 import com.trueaccord.scalapb.compiler.Version.scalapbVersion
 
-lazy val ScalaVersion = "2.11.8"
-lazy val ScalaVersions = Seq("2.11.8", "2.12.1")
+lazy val LanguageVersions = Seq("2.11.8", "2.12.1")
+lazy val LanguageVersion = sys.env.getOrElse("CI_SCALA_VERSION", LanguageVersions.head)
```

Comment:
You probably don't want to do this. I tried but hit on the problem that CI_SCALA_VERSION is 2.10 for ci-sbt-scalahost.